src/App.vue

```diff
@@ -78,7 +77,7 @@ const config = {
         {
             name: 'Hausnummer',
             identifier: 'housenumber',
-            type: 'Number',
+            type: 'number',
```

Comment:
just for your information: JS refers to Number and TS to number (without single quotes) if you want to display real data types and not string literals

Reply:
could be a huge improvement to make the query builder component type save...maybe in version 5.0

Reply:
would be great if you give me hints how to do this in the future ✌